definitions/npm/react-i18next_v7.x.x/flow_v0.64.x-/react-i18next_v7.x.x.js

```diff
@@ -0,0 +1,92 @@
+declare module "react-i18next" {
+  declare type TFunction = (key?: ?string, data?: ?Object) => string;
+
+  declare type TranslatorProps = {|
+    t: TFunction,
+    i18nLoadedAt: Date,
+    i18n: Object
+  |};
+
+  declare type TranslatorPropsVoid = {
+    t: TFunction | void,
+    i18nLoadedAt: Date | void,
+    i18n: Object | void
+  };
+
+  declare type Translator<P: {}, Component: React$ComponentType<P>> = (
+    WrappedComponent: Component
+  ) => React$ComponentType<
+    $Diff<React$ElementConfig<Component>, TranslatorPropsVoid>
+  >;
+
+  declare type TranslateOptions = $Shape<{
+    wait: boolean,
+    nsMode: "default" | "fallback",
+    bindi18n: false | string,
```

Comment:
@GAntoine @TLadd 
The casing of this option is incorrect, it should be: bindI18n

https://github.com/i18next/react-i18next/blob/114f9e0e123ed59f404e7b7eec9546797c728639/src/context.js#L4